### PR TITLE
Added hard coded directions

### LINKED ISSUE #104: In ResultsItemDetails Add directions <li> that takes an array of directions (prepare for working endpoint)

### PR DESCRIPTION
**Pull request by bretm9**
### Participating Group Members:

- @bretm9 

### Code Highlights:

- Added `Directions.vue`:
  - Added dynamically rendered list of directions based on `directions` property in `data`
  - Hard-coded directions for now, will need to be refactored to get directions from backend when endpoint is up.
- `ResultsItemDetails.vue` now renders `Directions.vue` component when `Get Directions` button is clicked.

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
N/A

### Message/Questions for reviewer:
N/A

### Issues:

- Addresses #17
- Closes #104 

### Screenshots (if appropriate):
Screen Shot 2021-01-12 at 12 32 53 PM

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: src/components/ResultsItemDetails.vue [hunk 1]
```diff
@@ -14,6 +14,11 @@
     </article>
     <button class='button-get-directions'>Get Directions</button>
     <!-- button is not functional yet - need to get a directions component w/ router -->
+    <section v-if="this.directions.length">
+      <section :key='i' v-for="(direction, i) in directions">
+        <h4>{{i+1}}. {{direction}}</h4>
```
Comment (nicolegooden) [inline src/components/ResultsItemDetails.vue:19]: I appreciate this logic, so that the user doesn't start viewing directions at Step 0.

**Review by nicolegooden — APPROVED** [review]
Nice Job, Bret!  Quick question: Are we still planning to make a directions component that takes the user to it if `Get Directions` is clicked?  

Just want to clarify if you're thinking we should render the directions directly in `ResultsItemDetails`.

File: src/components/ResultsItemDetails.vue (hunk 1 above)
Comment (stacyp2006) [reply src/components/ResultsItemDetails.vue:19]: I like this logic too, but I'm curious why you chose to do this over an ordered list?

**Review by stacyp2006 — APPROVED** [review]
Nice work! Excited to see this connected to the endpoint.

**Comment by bretm9** [comment]
@nicolegooden Oh ya we could move this logic into a separate component, but I figured the 'get directions' button would just populate the `directions` data in the `ResultsItemDetails` component so this would render. SRP would probably say we should put it into a separate component, so I can definitely refactor that later!

**Comment by bretm9** [comment]
I've updated the branch to include @stacyp2006's suggestion of using a `<ol>`, and @nicolegooden's suggestion of splitting the directions into it's own component. This should be good to merge now, but please review!

File: src/components/Directions.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<template>
+  <ol v-if="this.directions.length">
+    <li :key='i' v-for="(direction, i) in directions">
+      {{direction}}  
+    </li>
+  </ol>
+</template>
+
```
Comment (stacyp2006) [inline src/components/Directions.vue:8]: Nice! Good work on this!

**Review by stacyp2006 — APPROVED** [review]

**Comment by nicolegooden** [comment]
Thanks @bretm9 ! I was originally thinking we'd have a separate route for directions, that would render the directions component; however, it might make sense to just render the directions component as a child in `ResultsItemDetails`.  Which way do you think you'll go?

File: src/components/Directions.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<script>
+export default {
+  props: ["park"],
+  data() {
+    return {
+      directions: []
+    }
+  },
+  mounted() {
```
Comment (nicolegooden) [inline src/components/Directions.vue:17]: Thanks for adding these annotations.  It definitely makes things easier when we want to recall what we've already discussed, and where code should exist.

**Review by nicolegooden — APPROVED** [review]
Love this! Thanks for your hard work and flexibility with the design.

**Comment by nicolegooden** [comment]
I have resolved the merge conflict locally and pushed a new commit for that.  @npdarrington , feel free to merge this PR once you've gotten a chance to review!

**Review by npdarrington — APPROVED** [review]
Thanks for the update on this!

**Pull request merged**